dev-docs/modules/dfp_video.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Module - Dfp Video
+description: Addition of DFP Video to the Prebid package
+top_nav_section: dev_docs
+nav_section: modules
+module_code : dfpAdServerVideo
+display_name : Dfp Video
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# DFP Video Module
+{:.no_toc}
+
+This module is required to use the Prebid Instream video examples with DFP Adserver.
+
+## Instructions for adding Dfp Video code to a Prebid package
+
+### Step 1:  Prepare the base Prebid file as usual
+
+The standard options:
+
+- Build from a locally-cloned git repo 
+- Receive the email package from the Prebid [Download](http://prebid.org/download.html) page
+ 
+### Step 2: Integrate into your prebid.js configuration
+
+The method exposes the `pbjs.adServers.dfp.buildVideoUrl` method to use. See the Show Video Ads with DFP guide link below for a full example. 
```

Comment:
Would be nice for method name to link to API docs - if pressed for time, we can fix later.